CHANGES

```diff
@@ -16,6 +16,9 @@ Features added
 Bugs fixed
 ----------
 
+* #10031: pycode: Fix spurious whitespace in unparsing various operators (``+``,
```

Comment:
pycode is an internal module to parse python code. So it's better to use "py domain" instead of pycode. It's a feature name users face.
```suggestion
* #10031: py domain: Fix spurious whitespace in unparsing various operators (``+``,
```